core/kazoo_services/src/kz_services.erl

```diff
@@ -1272,12 +1274,18 @@ get_item_plan(CategoryId, ItemId, ServicePlan) ->
 %%--------------------------------------------------------------------
 -spec get_service_modules() -> atoms().
 get_service_modules() ->
-    case ?DEFAULT_SERVICE_MODULES of
-        'undefined' -> ?SERVICE_MODULES;
-        ConfModules ->
-            lager:debug("configured service modules: ~p", [ConfModules]),
-            [kz_term:to_atom(Mod, 'true') || Mod <- ConfModules]
-    end.
+    ServiceModules =
+        case kapps_config:get(?CONFIG_CAT, <<"modules">>) of
```

Comment:
What is this fixing?
* calls to typecasted kapps_config functions are much more prefered.
* you map kz_term:to_atom on a list of atoms in one of the 2 cases.